### DISCUSSION TITLE
How to highlite the current line

### DISCUSSION THREAD

**Discussion by froschi123** [ueberdosis/tiptap#1773] [General Questions]
I am working with the extension "extension-code-block-lowlight".
I want to high-lite the line where the cursor currently is. I have implemented a function to get the current line / row and the text from the current line. 
The code elements are separated by spans, but those do not allow to highlite the line, adding "div" does not work.
Any idea how to implement a line-highlite like it is done in most text / code editors for the current line?

**Comment by hanspagel** [comment]
I think you’d need support in lowlight for that feature and from what I know they don’t have support for it.

I’d love to see a CodeMirror extension for tiptap, that would bring a fully fledged code editor to tiptap, but it’s not there. I just found this community contributed CodeSandbox for React: https://codesandbox.io/s/tiptapcodemirror-0jiqt Maybe that helps.
